DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/CentreConfiguration/CentreConfigurationControllerTests.cs

```diff
@@ -82,6 +88,64 @@ public void EditCentreWebsiteDetails_should_redirect_to_error_page_when_API_issu
             result.Should().BeRedirectToActionResult().WithActionName("Error").WithControllerName("LearningSolutions");
         }
 
+        [Test]
+        public void EditCentreDetailsPostSave_with_signature_image_fails_validation()

```

Comment:
Also, "with signature image" sounds like a perfectly valid title to me. Is there some other wording that could be used for this test and the test below?